src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -340,14 +340,15 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
       case None => Put
     }
 
-    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } }
-
-    // don't leak any sensitive data
-    RequestComplete(NoContent)
-  }.recover {
+    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } } map { response =>
```

Comment:
If the call to rawls is truly bound to time out, then I think orchestration should return Accepted, and rawls should log success/failure.

The goal of all this work is that we are able to detect errors in the whitelist synchronization. We do this (as you know) via loggly alerts - those can look at any log. So as long as rawls logs failures, we can alert on the rawls failures - which may or may not require updates to the loggly alert criteria.

Orchestration can/should still log failures with everything except the call to rawls, of course.